l10n_it_fatturapa/views/partner_view.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_partner_form_fatturapa">
+        <field name="name">partner.form.fatturapa</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form" />
+        <field name="arch" type="xml">
+            <notebook position="inside">
+            <page
+                    name="fatturapa"
+                    string="Electronic Invoice"
+                    groups="account.group_account_invoice"
+                >
+                <group name="fatturapa_group">
+                    <group>
+                        <field name="electronic_invoice_no_contact_update" />
+                    </group>
+                    <group>
+                        <field name="electronic_invoice_subjected" />
+                        <field
+                                name="electronic_invoice_obliged_subject"
+                                attrs="{'invisible': [('electronic_invoice_subjected', '=', False)]}"
+                            />
+                    </group>
+                    <group
+                            attrs="{'invisible': ['|',('electronic_invoice_subjected', '=', False), ('electronic_invoice_obliged_subject', '=', False)]}"
+                        >
+                        <field
+                                name="ipa_code"
+                                placeholder="IPA123"
+                                attrs="{'invisible': [('is_pa','=', False)]}"
```

Comment:
Il campo `is_pa` non è presente nella vista, me ne sono accorto in un'installazione in cui Odoo solleva giustamente l'errore:
```
Field is_pa used in attrs.invisible ({'invisible': ['|',('is_pa', '=', True), ('codice_destinatario', '!=', '0000000')]}) must be present in view but is missing.
```
lo puoi aggiungere? Dovrebbe bastare inziccare da qualche parte:
```
<field name="is_pa" invisible="True"/>
```

Reply:
Aggiunto.